icu4c/source/i18n/unicode/timezone.h

```diff
@@ -204,10 +204,20 @@ class U_I18N_API TimeZone : public UObject {
      * all strings that createTimeZone() accepts)
      *
      * @return an enumeration object, owned by the caller.
-     * @stable ICU 2.4
+     * @deprecated ICU 70 Use createEnumeration(UErrorCode&) instead.
```

Comment:
Please surround each of the three deprecated functions with
```
#ifndef U_HIDE_DEPRECATED_API
...
#endif  // U_HIDE_DEPRECATED_API
```

... and that should fix the build bot failure.

Reply:
I see.. fixing now.

Reply:
done.